src/tito/tagger/main.py

```diff
@@ -243,10 +239,21 @@ def _make_changelog(self):
 
                 write(fd, header)
 
-                for cmd_out in output.split("\n"):
-                    write(fd, "- ")
-                    write(fd, "\n  ".join(textwrap.wrap(cmd_out, 77)))
+                # don't die if this is a new package with no history
+                if self._changelog_msg is not None:
+                    write(fd, self._changelog_msg)
```

Comment:
Could be single or multiline, I was going with the idea that if you are providing your own changelog message you don't want tito doing any assumed formatting. 

I could add '--changelog-file' as another option for those that have a file or want to write to a file first. In my current usage, I have a script that analyzes a git repo, pulls out commit messages, formats them the way I want and then calls `tito tag --changelog="MYMSG"` so enforcing a file for me would just cause me to have to write a file I don't need.